catalyst_voices/packages/internal/catalyst_voices_blocs/lib/src/campaign_status/campaign_status_state.dart

```diff
@@ -0,0 +1,22 @@
+part of 'campaign_status_cubit.dart';
+
+enum CampaignStatus {
+  draft,
+  published;
+
+  bool get isDraft => this == draft;
+  bool isSelected(CampaignStatus? selected) => this == selected;
```

Comment:
Its confusing name in my opinion, this just equals and doesn't mean its selected. Maybe it one of widget it does mean it but well, just there. should it be here?